### DISCUSSION TITLE
Possible to parse Bounce reports?

### DISCUSSION THREAD

**Discussion by chetanmadaan** [php-mime-mail-parser/php-mime-mail-parser#365] [General]
Hi,

Thanks for the very useful wrapper. Works really well (so far).

Let me take a minute to explain my setup. We are sending out emails using SendGrid and any bounces go to the from address, which go to Amazon SES, which then sends over a webhook with the copy of the message. 

I am trying to setup a webhook which can read the message but all the information, I am able to receive is for the message that was received from SendGrid (not the original message which was sent, which is embedded in the email).

Does this make sense or am I going Crazy here?

**Comment by eXorus** [comment]
Hello,

Sorry I don't understand, do you have an example of the message received?
